.gitignore

```diff
@@ -82,3 +82,17 @@ samples/affinidi-dotnet-openidconnect/obj/Debug/net8.0/staticwebassets/msbuild.A
 samples/affinidi-dotnet-openidconnect/obj/Debug/net8.0/staticwebassets/msbuild.build.Affinidi-Login-Demo-App.props
 samples/affinidi-dotnet-openidconnect/obj/Debug/net8.0/staticwebassets/msbuild.buildMultiTargeting.Affinidi-Login-Demo-App.props
 samples/affinidi-dotnet-openidconnect/obj/Debug/net8.0/staticwebassets/msbuild.buildTransitive.Affinidi-Login-Demo-App.props
+generator/dotnet/template/obj/Affinidi-Login-Demo-App.csproj.nuget.g.props
```

Comment:
Can we use a `*` to filter multiple resources instead of specifying those separately?

P.S. there are duplicates in this file

Reply:
modified to ignore the directory instead of individual files.